src/DotNetEnv/LoadOptions.cs

```diff
@@ -4,47 +4,54 @@ namespace DotNetEnv
 {
     public class LoadOptions
     {
-        public static readonly LoadOptions DEFAULT = new LoadOptions();
-
-        public bool SetEnvVars { get; }
-        public bool ClobberExistingVars { get; }
-        public bool OnlyExactPath { get; }
+        public bool SetEnvVars { get; set; }
+        public bool ClobberExistingVars { get; set; }
+        public bool OnlyExactPath { get; set; }
 
         public LoadOptions(
             bool setEnvVars = true,
             bool clobberExistingVars = true,
             bool onlyExactPath = true
-        ) {
+        )
+        {
             SetEnvVars = setEnvVars;
             ClobberExistingVars = clobberExistingVars;
             OnlyExactPath = onlyExactPath;
         }
 
-        public LoadOptions(
-            LoadOptions old,
-            bool? setEnvVars = null,
-            bool? clobberExistingVars = null,
-            bool? onlyExactPath = null
-        ) {
-            SetEnvVars = setEnvVars ?? old.SetEnvVars;
-            ClobberExistingVars = clobberExistingVars ?? old.ClobberExistingVars;
-            OnlyExactPath = onlyExactPath ?? old.OnlyExactPath;
-        }
+        public static LoadOptions DEFAULT => new LoadOptions();
+
+        public static LoadOptions NoEnvVars(LoadOptions options = null) =>
+            (options ?? DEFAULT).NoEnvVars();
+
+        public static LoadOptions NoClobber(LoadOptions options = null) =>
+            (options ?? DEFAULT).NoClobber();
 
-        public static LoadOptions NoEnvVars (LoadOptions options = null) =>
-            options == null ? DEFAULT.NoEnvVars() : options.NoEnvVars();
+        public static LoadOptions TraversePath(LoadOptions options = null) =>
+            (options ?? DEFAULT).TraversePath();
+    }
 
-        public static LoadOptions NoClobber (LoadOptions options = null) =>
-            options == null ? DEFAULT.NoClobber() : options.NoClobber();
+    public static class LoadOptionExtensions
+    {
+        public static LoadOptions NoEnvVars(this LoadOptions options)
```

Comment:
These require mutability, which makes the code ever so slightly cleaner, perhaps, but at the cost of removing immutability, which makes it more dangerous, imo.